src/components/Label/Label.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {StyleSheet, Text} from 'react-native';
+import {fontStyles} from '../../constants';
+import {useThemeColor} from '../../hooks';
+import {LabelProps} from './Label.types';
+
+export const Label = (props: LabelProps) => {
```

Comment:
Every component should have stories. Please add for this

Reply:
Added.